third_party/move/move-bytecode-verifier/src/check_duplication.rs

```diff
@@ -278,6 +327,20 @@ impl<'a> DuplicationChecker<'a> {
         }
     }
 
+    fn check_duplicate_variants<'l>(
```

Comment:
Can we add a test for this?

Reply:
See above. This PR will not boil the ocean to add tests where there weren't much before.